File: EIMS/EIMS_UI/ClientApp/src/app/models/view-models/shift-view-model.ts
import { ShiftType } from '../common/constants-model';

export class ShiftViewModel {
  constructor(
    public shiftId?: number,
    public shiftType?: ShiftType,
    public startTime?: Date | string,
    public endTime?: Date | string,
    public active?: boolean
  ) { }
}

File: EIMS/EIMS_Lib/ViewModels/ShiftViewModel.cs
using EIMS_Lib.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EIMS_Lib.ViewModels
{
    public class ShiftViewModel
    {
        public int ShiftId { get; set; }
        public ShiftType ShiftType { get; set; }
      
        public DateTime StartTime { get; set; }
   
        public DateTime EndTime { get; set; }
        
        public bool Active { get; set; }
    }
}
